Mathlib/Topology/SubsetProperties.lean

```diff
@@ -128,10 +128,10 @@ theorem IsCompact.diff (hs : IsCompact s) (ht : IsOpen t) : IsCompact (s \ t) :=
 #align is_compact.diff IsCompact.diff
 
 /-- A closed subset of a compact set is a compact set. -/
-theorem isCompact_of_isClosed_subset (hs : IsCompact s) (ht : IsClosed t) (h : t ⊆ s) :
+theorem IsCompact.of_isClosed_subset (hs : IsCompact s) (ht : IsClosed t) (h : t ⊆ s) :
```

Comment:
You've pointed out that you're not sure that the new dot-notation improved readability. I think that one reason for this is that this lemma proves a property of `t`, but the dot-notation takes as argument a property of `s`.
If this is renamed to `IsClosed.isCompact_of_subset`, then you would instead get dot-notation of the form `ht.isCompact_of_subset hs h`. I conjecture that this will improve readability.

Reply:
FWIW I think both names are reasonable. I don't have a strong preference either way.

Reply:
To be honest, I'm not sure if the other is better. I'll leave it as-is, unless you feel strongly.

(To be clear: I think the new dot notation is better in many cases; I just was unsure about some instances. If the receiver is short, dot notation is a clear win to me, resembling method chaining in programming languages. For a long term in parentheses, I'm less sure. At some point, *I* find dot notation less readable.)